### PR TITLE
Bugfix/camera inactive warning message not displaying fallback

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem

On some integrations where the SDK configuration does not have `uploadFallback` defined in the Document step, the Live Document Capture UI's warning popup when camera inactivity is detected does not display the basic camera mode fallback as expected when `uploadFallback` is not disabled/defined.

# Solution

Pass a default value for `DocumentLiveCapture` component's `isUploadFallbackDisabled` prop if `uploadFallback` is not defined.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Capture/Document.tsx [hunk 1]
```diff
@@ -229,7 +229,7 @@ class Document extends Component<Props> {
         <DocumentLiveCapture
           containerClassName={style.liveDocumentContainer}
           documentType={documentType}
-          isUploadFallbackDisabled={!uploadFallback}
+          isUploadFallbackDisabled={uploadFallback ? !uploadFallback : false}
```
Comment (stefaniacardenas) [inline src/components/Capture/Document.tsx:232]: This statement always returns `false` 🤔

Comment (kopijunkie) [reply src/components/Capture/Document.tsx:232]: Ah, you're right. & there's a simpler way of doing what I'm trying to achieve here. Will fix.

File: src/components/Capture/Document.tsx [hunk 2]
```diff
@@ -173,7 +173,7 @@ class Document extends Component<Props> {
       side,
       trackScreen,
       translate,
-      uploadFallback,
+      uploadFallback = false,
```
Comment (stefaniacardenas) [inline src/components/Capture/Document.tsx:176]: So by default we want `uploadFallback` to be `false`? I believe the default for selfie is true

Comment (kopijunkie) [reply src/components/Capture/Document.tsx:176]: That's true. I've confused this `uploadFallback` default value with `isUploadFallbackDisabled` 🙈

**Review by stefaniacardenas — APPROVED** [review]

**Pull request merged**